packages/gatsby/src/commands/develop.ts

```diff
@@ -372,8 +375,6 @@ module.exports = async (program: IProgram): Promise<void> => {
   const files = [rootFile(`gatsby-config.js`), rootFile(`gatsby-node.js`)]
   let lastConfig = requireUncached(rootFile(`gatsby-config.js`))
 
-  let watcher
```

Comment:
I see that we never assigned `watcher` (I assume that was meant for `chokidar.watch`), but instead of removing it should we try to actually assign it properly and close it?

Reply:
I don't know if chokidar needs it, I guess chokdiar would cleanup anyway but good idea

Reply:
I found this commit in chokidar repo - https://github.com/paulmillr/chokidar/commit/89f5239e02ca9733adb5b211d15924573a04e611 which is somewhat related, but also not exactly - I guess for that commit it's more about clearing out Node's event loop so it's able just exit itself without forcing it with something like `process.exit`